javascript/javascript2/week1/danish-letters-finder.js

```diff
@@ -0,0 +1,31 @@
+function findDanishLetters(str) {
+    let countA = 0;
```

Comment:
the variable names are not okay...they mean nothing. Also you could think to use an object that has as keys the danish characters and the values as the number of their occurences. E.g `danishCharacters = {æ:0 , å:0, ø:0  }`
```suggestion
    let countA = 0;
```

Reply:
This homework is also implemented. Do you have time to review it?